data/json/furniture.json

```diff
@@ -3381,5 +3381,207 @@
       "ter_set": "t_floor",
       "items": [ { "item": "pillow", "count": 19 }, { "item": "blanket", "count": 3 } ]
     }
+  },
+  {
+    "type": "furniture",
+    "id": "f_file_cabinet",
+    "name": "filing cabinet",
+    "description": "A set of drawers in a sturdy metal cabinet, used to hold files.  It can be locked to protect important information.",
+    "symbol": "}",
+    "color": "dark_gray",
+    "move_cost_mod": 2,
+    "required_str": 8,
+    "flags": [ "PLACE_ITEM", "TRANSPARENT", "CONTAINER", "BLOCKSDOOR", "MOUNTABLE" ],
+    "deconstruct": {
+      "items": [
+        { "item": "sheet_metal", "count": [ 2, 6 ] },
+        { "item": "scrap", "count": [ 2, 6 ] },
+        { "item": "clockworks", "count": [ 0, 2 ] }
```

Comment:
Why are these here?

Reply:
There is no lock item, but the lock on the filing cabinet should be exactly the kind of lock you can salvage to make a cargo lock. This is the best I can do to represent that.

Reply:
I don't think this is an appropriate solution. Either add a specific item for a lock or remove the clockworks from the items table.

Reply:
I hadn't aimed to add a bunch of new lock stuff, but sure, what's one more tangent... ;)